docs/app-dev/getting-started.md

```diff
@@ -17,10 +21,13 @@ using Tendermint.
 ### Install
 
 The first apps we will work with are written in Go. To install them, you
-need to [install Go](https://golang.org/doc/install) and put
-`$GOPATH/bin` in your `$PATH`; see
-[here](https://github.com/tendermint/tendermint/wiki/Setting-GOPATH) for
-more info.
+need to [install Go](https://golang.org/doc/install), put
+`$GOPATH/bin` in your `$PATH` and enable go modules with these instructions:
+```bash
+echo export GOPATH=\"\$HOME/go\" >> ~/.bash_profile
```

Comment:
I **don't believe** newer Golang versions require setting GOPATH, but that's fine for now